responses/content.js

```diff
@@ -43,16 +43,31 @@ function theAnswerGivenWasCorrect(articleData, newQuestion){
 
 }
 
-function theAnswerGivenWasNotCorrect(expectedAnswer, articleData){
+function theAnswerGivenWasNotCorrect(expectedAnswer, articleData, scoreData){
 	const displayPhrase  = `Sorry, that is incorrect. The correct answer was ${expectedAnswer.replace('people:', '')}. They were connected in the FT article:`;
 	const voicePhrase = `Sorry, that is incorrect. The correct answer was ${expectedAnswer.replace('people:', '')}. They were connected in the FT article, titled: ${articleData.title}.`;
+	let scorePhrase = `You made ${scoreData.score} connection${ (parseInt(scoreData.score)!== 1)?'s':'' }.`;
+
+	if(parseInt(scoreData.score) >= parseInt(scoreData.scoreMax)) {
```

Comment:
We're cast scoreData.score to an integer quite a few times. Why don't we declare a variable (which has been casted) at the start of the method and use that throughout the rest of the code?